scripts/quests/ahtUrhgan/Saga_of_the_Skyserpent.lua

```diff
@@ -96,9 +96,7 @@ quest.sections =
             onZoneIn =
             {
                 function(player, prevZone)
-                    if quest:getVar(player, 'Prog') == 1 then
-                        return 12
-                    end
+                    return 12
```

Comment:
Are players supposed to get this CS repeatedly? Why remove this prog check?

Reply:
isn't the condition already checked on line 74?

Reply:
oh most certainly! Didn't see that with the small changelog window.